src/js/pages/home.js

```diff
@@ -1,22 +1,21 @@
 import api from '../api';
 import { renderCards } from '../categories/categories-api';
-import { handleClick, addListener } from '../../main';
+import { handleClickFilter } from '../../main';
```

Comment:
handleClick should be imported from main and modified here if needed

Reply:
fixed